interface/weno/admin.php

```diff
@@ -50,11 +56,13 @@
     print xlt("Weno Service is Enabled")."<br><br>";
 }
 
-$drugData = $tables->drugTableInfo();
-if (!$drugData['ndc']) {
-    echo "<a href='drugPaidInsert.php?csrf_token_form=" . attr_url(collectCsrfToken()) . "' class='btn btn-default'>".xlt("Import Formularies")."</a> <br>".xlt("Be patient, this can take a while.");
-} else {
-    print xlt("Formularies inserted into table")."<br>";
+if ($tableDoesExist == true && $tableHasData == true) {
```

Comment:
isn't the table always going to exist?
To see if there is data in the tables, would just use query (no need to use those upgrade functions):
```
SELECT count(drug_id) FROM erx_weno_drugs
```